lib/rpm_package.py

```diff
@@ -130,3 +130,11 @@ def _load(self):
                     distro_version=self.distro.version)
         except TypeError:
             raise exception.PackageDescriptorError(package=self.name)
+
+    @property
+    def version(self):
+        return self.spec_file.query_tag("version")
```

Comment:
We will start a process and open the file every time we check this property. I think we should cache this value.